crates/router/src/core/payments/flows/session_flow.rs

```diff
@@ -441,6 +479,7 @@ fn get_session_request_for_simplified_apple_pay(
 
 fn get_session_request_for_manual_apple_pay(
     session_token_data: payment_types::SessionTokenInfo,
+    merchant_domain: Option<String>,
 ) -> RouterResult<payment_types::ApplepaySessionRequest> {
     let initiative_context = session_token_data
```

Comment:
this will be removed in future once we stop allowing domain configuration from dashboard.

Reply:
Yeah, it will be removed once the new changes to consume the domain from header is working fine.